src/App.jsx

```diff
@@ -15,8 +16,21 @@ import Applications from "./pages/Applications";
 import ResetPassword from "./pages/ResetPassword";
 import ForgetPassword from "./pages/ForgetPassword";
 import PostJob from "./pages/PostJob";
+import useUserStore from "./store/user-store";
 
 function App() {
+  const { fetchSeeker } = useUserStore(); // Destructure the fetchSeeker function from the user store
+
+  useEffect(() => {
```

Comment:
we probably don't need this useEffect anymore, just comment out in case

Reply:
That would mean the fetch would only run during login.

It wont be able to fetch when you refresh the page for example.

I'll check to see if we need the fetch function when logging in instead

Reply:
> ah okay gotchu, maybe that's why i wasn't able to refresh the navbar mb

scratch this comment for the useEffect it fixed my navbar 💀